bin/docker-setup.sh

```diff
@@ -14,34 +14,32 @@ redirect_output() {
     fi
 }
 
-# --user 33:33 forces the wordpress:cli container to use a user with the same ID as the main wordpress container. See:
-# https://hub.docker.com/_/wordpress#running-as-an-arbitrary-user
-# https://github.com/docker-library/wordpress/discussions/909#discussioncomment-9941634
-cli()
+wp_cli()
 {
 	INTERACTIVE=''
 	if [ -t 1 ] ; then
 		INTERACTIVE='-it'
 	fi
-	redirect_output docker run $INTERACTIVE --env-file default.env --rm --user 33:33 --volumes-from $WP_CONTAINER --network container:$WP_CONTAINER wordpress:cli "$@"
+
+	redirect_output docker exec $INTERACTIVE --env-file default.env --user www-data $WP_CONTAINER wp "$@"
```

Comment:
Couple of things I noticed

1. `docker exec` doesn't accept `--env` argument. It's used with `docker run`
2. The `wp` prefix can be avoided because it's always ideal for us to provide full WP CLI commands. With this refactoring, we're adding commands without the wp prefix, which kinda reduces code readability. If you have a strong preference to this approach, I'm ok with keeping it.

Ideally I would prefer keeping the wrapper method name as `cli`, so you don't need to do too many refactorings.

Reply:
> docker exec doesn't accept --env argument. It's used with docker run

I see [it does support both `--env` and `--env-file` since API v1.25](https://docs.docker.com/reference/cli/docker/container/exec/).

> The wp prefix can be avoided because it's always ideal for us to provide full WP CLI commands. With this refactoring, we're adding commands without the wp prefix, which kinda reduces code readability. If you have a strong preference to this approach, I'm ok with keeping it.

I have no preference either way. I am going to trust your judgment on this; I made [the changes](https://github.com/Automattic/woocommerce-payments/pull/9256/commits/03f18833687acce66da704ad55bb39bbd88b5065).